lib/runners/schema/result.rb

```diff
@@ -19,7 +19,8 @@ module Schema
       let :warning, object(message: string, file: string?)
       let :analyzer, object(name: string, version: string)
 
-      let :success, object(guid: string, timestamp: string, type: literal("success"), issues: array(Issue.issue), analyzer: analyzer)
+      let :success, enum(object(guid: string, timestamp: string, type: literal("success"), issues: array(Issue.issue), analyzer: analyzer),
+                         object(guid: string, timestamp: string, type: literal("success"), analyzer: analyzer))
```

Comment:
[question] Cannot we simply make `issues` optional?

```suggestion
      let :success, object(guid: string, timestamp: string, type: literal("success"), issues: array?(Issue.issue), analyzer: analyzer)
```

Reply:
Of course, yes. I chose `enum` to completely remove the `issues` from the JSON.

Reply:
OK, let's keep it as-is. 👍